### DISCUSSION TITLE
Viscous darkness in Azimuth ballast (ballast goblins presence?)

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#15203] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/user-attachments/assets/4dd784d0-9988-4839-91c8-064d535b1002

### Reproduction steps

Here:
![Here](https://github.com/user-attachments/assets/ab5142da-e2e7-4878-9967-7bd5c7f53dc7)

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
2 more cases:
Dugong engine room
![20241203232402_1](https://github.com/user-attachments/assets/94372bc8-b835-4e2a-b72e-1ba2bc772158)
![20241203232354_1](https://github.com/user-attachments/assets/c886679a-f9f0-4bac-9ac0-b89120d634ce)
A little one in Kastrull drone
![20241203233141_1](https://github.com/user-attachments/assets/03bfffc3-9fcb-40f5-83e8-8d932847555a)

This looks like an edge case, some very unlucky arrangement of walls, 1pixel to the right/left and it won't happen

https://github.com/user-attachments/assets/670e239f-f3dc-439b-b32c-b4f40bb6a332

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/2e877aa3e87ee418c12bc3fbc506eb75989a2b2e (commit in our private repo). The fix didn't make it in time for the winter update, but coming up in the next one.
